forestserviceprototype/specialuseform/models.py

```diff
@@ -3,49 +3,164 @@
 from django.contrib.auth.models import User
 from localflavor.us import models as localflavor
 
+
 # Create your models here.
 class Permit(models.Model):
```

Comment:
It should be `permit.id`, the default. I didn't see a reason to replace it, but if there's a special format for the field I'm happy to add it back and document that.